packages/appstore/src/components/transaction-list/transactions-for-a-day/transactions-for-a-day.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { useStore } from '@deriv/stores';
+import { useActiveWallet, useWalletTransactions } from '@deriv/hooks';
+import CompletedTransaction from '../completed-transaction';
+
+const TransactionsForADay = ({
+    day,
+    transaction_list,
+}: {
+    day: string;
+    transaction_list: ReturnType<typeof useWalletTransactions>['transactions'];
```

Comment:
Maybe would be better to move component props type above the component as you did with 2 other components above? It's just a suggestion, you can ignore this)

Reply:
Done, thx